### DISCUSSION TITLE
No PDF found (0.2.1)

### DISCUSSION THREAD

**Discussion by amz71** [mkb79/audible-cli#112] [Q&A] [answered]
"No PDF found for..." is shown for every file, even when a pdf is available in the library.

Any idea? Looks like the call to get_pdf_url always returns none, as pdf_url is not initialized (in _prepare_item?) ?

**Comment by mkb79** [comment, marked as answer]
Thank you for this information. Fixed this in the latest master. Can you try it out?

**Comment by amz71** [comment]
Perfect! Working now.
